server/api/users/models.py

```diff
@@ -55,8 +55,8 @@ class User(AbstractUser):
     team_signup = models.BooleanField(default=False)  # boolean
     has_consent = models.BooleanField(default=False)  # boolean
     travel_method = models.CharField(max_length=100, blank=True)
-    subteam_id = models.ForeignKey(SubTeam, null=True, on_delete=models.CASCADE)
-    team_id = models.ForeignKey(Team, null=True, on_delete=models.CASCADE)
+    subteam_id = models.ForeignKey(SubTeam, null=True, blank=True, on_delete=models.CASCADE)
+    team_id = models.ForeignKey(Team, null=True, blank=True, on_delete=models.CASCADE)
     team_admin = models.BooleanField(default=False)
```

Comment:
Should the FK be on_delete=models.CASCADE for optional params? I think this implies if the team is deleted, the user will also be deleted. Same thing for other optional params. 
I did it like this in events: team_id = models.ForeignKey(Team, on_delete=models.SET_NULL, null=True), so if the team is deleted, the team_id is just null for the event.

Reply:
Yes looks like it should definitely be SET_NULL. I didn't do this but I will push the change.